docs(traffic_shaping): the plugin is always enabled (#4953)
apollographql/router: docs/source/configuration/traffic-shaping.mdx

## Configuration

> The `traffic_shaping` plugin is always enabled with default values, if you want to override default values you should add `traffic_shaping` to your [YAML config file](./overview/#yaml-config-file), like so:

Reviewer: Good call, @garypen. It would be great if we could have a table of defaults somewhere on this page that could be linked like so:

```suggestion
By default, the `traffic_shaping` plugin is enabled with [preset values](#link-to-default-values). To override presets, add `traffic_shaping` to your [YAML config file](./overview/#yaml-config-file) like so:
```

      experimental_http2: enable # Configures HTTP/2 usage. Can be 'enable' (default), 'disable' or 'http2only'
```

> ### Preset values
> 
> + `timeout`: all timeouts are set at 30 secs by default.
> + `experimental_http2`: enabled by default

Reviewer 2: ```suggestion
### Preset values

The preset values of `traffic_shaping` that's enabled by default:

- `timeout: 30s` for all timeouts
- `experimental_http2: enable`
```
